src/utils/deploy.ts

```diff
@@ -241,3 +241,8 @@ const buildApiOptions = (opts: Partial<DeployOptions>): MetadataApiDeployOptions
   ...(opts['test-level'] ? { testLevel: opts['test-level'] } : {}),
   purgeOnDelete: opts['purge-on-delete'] ?? false,
 });
+
+export function buildDeployUrl(deployId: string): string {
+  const orgInstanceUrl = OrgConfigProperties.ORG_INSTANCE_URL;
```

Comment:
I think you'll need to get the instance URL from the org instead since `OrgConfigProperties.ORG_INSTANCE_URL` is a constant for a field:


`org.getField(Org.Fields.INSTANCE_URL);`

So maybe `buildDeployUrl` takes an `Org` type arg as well.  e.g.,
`export function buildDeployUrl(org: Org, deployId: string): string`

and a command would pass in the org flag value: `flags['target-org']`

Reply:
Thanks for the feedback. Just updated it to accept an org input and use the getField function.

I had to revert the changes I made to the "resume" command since I'm not sure how to proceed with passing the Org for it.

Reply:
Probably move that out of the conditional (above).  We can run tests with it as is and see what things look like.  We can add it to resume in another PR if needed.